packages/local_auth/lib/local_auth.dart

```diff
@@ -72,4 +74,41 @@ class LocalAuthentication {
     }
     return await _channel.invokeMethod('authenticateWithBiometrics', args);
   }
+
+  /// Returns true if device is capable of checking biometrics
+  ///
+  /// Returns a [Future] bool true or false:
+  Future<bool> canCheckBiometrics() async {
+    final List<dynamic> biometrics =
+        await _channel.invokeMethod('getAvailableBiometrics');
+    return biometrics.isNotEmpty;
```

Comment:
Can you shorten this to:

```dart
Future<bool> get canCheckBiometrics async => (await _channel.invo.....).isNotEmpty;
```

I would like to avoid putting `dynamic` in this code.